src/sys/ptrace/linux.rs

```diff
@@ -304,12 +304,15 @@ fn ptrace_peek(
     }
 }
 
-/// Get user registers, as with `ptrace(PTRACE_GETREGS, ...)`
+/// Get user registers, as with `ptrace(PTRACE_GETREGS, ...)`. Call [`getregset`] for safe version
 ///
 /// Note that since `PTRACE_GETREGS` are not available on all platforms (as in [ptrace(2)]),
 /// `ptrace(PTRACE_GETREGSET, pid, NT_PRSTATUS, ...)` is used instead to achieve the same effect
 /// on aarch64 and riscv64.
 ///
+/// Currently, in x86_64 platform, if the tracer is 64bit and tracee is 32bit, the return value is
+/// undefined.
+///
```

Comment:
The docs should have a `# Safety` section.

Reply:
Updated